src/epd3in7/mod.rs

```diff
@@ -0,0 +1,263 @@
+//! A simple Driver for the Waveshare 3.7" E-Ink Display via SPI
+//!
+//!
+//! Build with the help of documentation/code from [Waveshare](https://www.waveshare.com/wiki/3.7inch_e-Paper_HAT),
+use embedded_hal::{
+    blocking::{delay::DelayMs, spi::Write},
+    digital::v2::{InputPin, OutputPin},
+};
+
+pub(crate) mod command;
+mod constants;
+
+use self::command::Command;
+use self::constants::*;
+
+use crate::buffer_len;
+use crate::color::Color;
+use crate::interface::DisplayInterface;
+use crate::traits::{InternalWiAdditions, RefreshLut, WaveshareDisplay};
+
+/// Width of the display.
+pub const WIDTH: u32 = 280;
+
+/// Height of the display
+pub const HEIGHT: u32 = 480;
+
+/// Default Background Color
+pub const DEFAULT_BACKGROUND_COLOR: Color = Color::White;
+
+const IS_BUSY_LOW: bool = false;
+
+/// Display with Fullsize buffer for use with the 2in9 EPD
```

Comment:
2in9->3in7

It doesn't let me make code suggestions .

Reply:
isn't this your code ?

Reply:
lol, probably, I thought it was introduced in the changes afterwards.

Reply:
I was able to make the code suggestion that you wanted.